include/mbedtls/config_adjust_legacy_crypto.h

```diff
@@ -104,7 +104,6 @@
 #if defined(MBEDTLS_PSA_CRYPTO_C)
 
 #if defined(MBEDTLS_PSA_ACCEL_ALG_MD5)
-#define MBEDTLS_MD_CAN_MD5
```

Comment:
In this PR we remove the MBEDTLS_MD_CAN_xxx macros. They are referenced in psa-migration/md-cipher-dispatch.md and psa-migration/strategy.md but in the context of 4.0 I am not sure if we should update them. @gilles-peskine-arm and @mpg as the main authors of these documents do you think we should update them in this PR?

MBEDTLS_MD_CAN_xxx is also in docs/driver-only-builds.md and I think we should update that file.

Reply:
We should definitely update the documentation. I think it'll be easier to do it later, after we've completed the configuration rework, rather than one symbol at a time.

Reply:
okay thanks, I will create an issue to not forget about it then.

Reply:
> docs/driver-only-builds.md can be updated in this PR I think.

I see it was done and I think that's good, but since you're creating an issue for psa-migration/md-cipher-dispatch.md and psa-migration/strategy.md can you also add a note there to do a final pass on docs/driver-only-builds.md as well just to ensure everything's consistent?

Reply:
@gilles-peskine-arm @mpg I've created https://github.com/Mbed-TLS/mbedtls/issues/9626. Which project I should put that issue into?

Reply:
Thanks. That's technically a 4.0 thing (even though we've otherwise treated the configuration migration separately from 4.0 planning), so if you don't have a good way to put it in your planning, we can put it in the “needs to be done near the end” column of the 4.0 planning board.

Reply:
Thanks. I've added it there then.